README.md

```diff
@@ -260,6 +260,7 @@ Table of Contents
   * [typetalk.com](https://www.typetalk.com/) — Share and discuss ideas with your team through instant messaging on the web or on your mobile
   * [Tugboat](https://tugboat.qa) - Preview every pull request, automated and on-demand. Free for all, complimentary Nano tier for non-profits.
   * [whereby.com](https://whereby.com/) — One click video conversations, for free (formerly known as appear.in)
+  * [userforge.com](https://userforge.com/) - Interconnected online personas, user stories and context mapping.  Helps to keep design and dev in sync.
```

Comment:
thanks, please add something liike `free for up to 3 personas and 2 collaborators` to the description

Reply:
Done!